src/Documentation/fpga/xrt.rst

```diff
@@ -566,22 +608,24 @@ Every partition also exports its UUIDs. See below for examples::
 hwmon
 -----
 
-xmgmt driver exposes standard hwmon interface to report voltage, current, temperature,
-power, etc. These can easily be viewed using *sensors* command line utility.
+xmgmt driver exposes standard hwmon interface to report voltage, current,
+temperature, power, etc. These can easily be viewed using *sensors* command
+line utility.
 
 
 mailbox
 -------
 
-xmgmt communicates with user physical function driver via HW mailbox. Mailbox opcodes
-are defined in ``mailbox_proto.h``. `Mailbox Inter-domain Communication Protocol
-<https://xilinx.github.io/XRT/master/html/mailbox.proto.html>`_ defines the full
-specification. xmgmt implements subset of the specification. It provides the following
-services to the UPF driver:
+xmgmt communicates with user physical function driver via HW mailbox. Mailbox
+opcodes are defined in ``mailbox_proto.h``.
+`Mailbox Inter-domain Communication Protocol <https://xilinx.github.io/XRT/master/html/mailbox.proto.html>`_
+defines the full specification. xmgmt implements subset of the specification.
+It provides the following services to the UPF driver:
 
-1.  Responding to *are you there* request including determining if the two drivers are
-    running in the same OS domain
-2.  Provide sensor readings, loaded xclbin UUID, clock frequency, shell information, etc.
+1.  Responding to *are you there* request including determining if the two drivers
```

Comment:
As for the part of "determining if both drivers are in the same OS domain", I'm thinking that we probably would remove the support of this from our new driver. It is not safeThis mechanism is for xocl to determine if it should send xclbin download request op code 7 (just pointer) or 8 (the entire xclbin). But, it is not safe for xmgmt to determine if it should accept op code 7 solely by this mechanism (the configuration of channel switch should be used to determine if 7 is safe op code to process by xmgmt in public cloud environment). I'm worrying that if this will cause some confusion and asking for unnecessary review questions?

Reply:
agree, will update